.github/workflows/test-workflow.yml

```diff
@@ -0,0 +1,41 @@
+name: Test Frontend and backend
+# This workflow is triggered on pushes to the repository.
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test-frontend:
+    name: Test ember frontend
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository files
+        uses: actions/checkout@v2
+      - name: Install node js
+        uses: actions/setup-node@v1
+        with:
+          node-version: '12.x'
+      - name: Install node dependencies
+        run: yarn install
+      - name: Run frontend tests
+        run: yarn test
+  test-backend:
+    name: Test php backend
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository files
+        uses: actions/checkout@v2
+      - name: Install php
+        uses: shivammathur/setup-php@v1
+        with:
+          php-version: '7.4'
```

Comment:
We would need to run the API tests for all supported PHP versions. Currently that's 7.1 to 7.4. I'm fine with dropping support for 7.1 cause we are still doing prereleases and 7.1 is not supported officially. But we should test for 7.2 - 7.4.

Reply:
ok. should be possible with matrix tests. There was even a example at the php action. I will check it out.